main.py

```
@@ -123,18 +172,30 @@ def _crawl_and_convert(self):
                     time.sleep(request_delay)
                     
             except Exception as e:
+                error = FileSystemError(
+                    message=f"ページ処理中の予期しないエラー",
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)